frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardView.stories.tsx

```diff
@@ -143,7 +151,7 @@ function createDashboard({ hasScroll }: CreateDashboardOpts = {}) {
           name: "Table",
           display: "table",
         }),
-        ...(!hasScroll ? { col: 12 } : {}),
+        ...(!hasScroll ? { col: 12 } : { row: 8 }),
```

Comment:
This change fixes the footer layout, since it seems it's used tot calculate the dashboard footer location.